### ISSUE TITLE
Avatar and Photo...

### ISSUE THREAD

**Issue by Eternity512** [wireapp/wire-desktop#1283]
Wire version: 3.0.28.28
Wire for web version: 2018.02.16.0754
Operating system: Windows 7 x64 
Which antivirus software do you have installed: Anti-Virus is not installed in the system

1. The photos you send and receive are not displayed.
2. User avatars are not displayed.

Solution
- Enable VPN

Every time I turn on the VPN, everything is displayed. But this does it every time fed up(

P.S. The system is installed by Windows Firewall Controll. Deactivate Windows Firewall Controll and rebooted the system. File hosts clean. Also checked the system HitmanPro and Emsisoft Emergency Kit. Programs are portable. Everything is clear. 

![2018-02-18_104819](https://user-images.githubusercontent.com/16630720/36349853-1170f5b4-1485-11e8-9804-ba0e2b3d5461.png)
![2018-02-18_111317](https://user-images.githubusercontent.com/16630720/36349854-16d49556-1485-11e8-9df9-6474cfbc2d60.png)

**Comment by Eternity512** [comment]
The problem is solved! Removed the rules from the Windows firewall that were added to block the telemetry of Windows. After removing the rules, pictures and avatars are displayed. Sorry for disturbing you.

**Comment by Eternity512** [comment]
the problem returned(

**Comment by Eternity512** [comment]
I again solved this problem  :D
I had a service installed to bypass the provider's locks, which can block packets with redirection from passive DPI, replace Host with hoSt, remove the space between the colon and host value in the Host header, "fragment" HTTP and HTTPS packets (install TCP Window Size) , and add an extra space between the HTTP method and the path.

Disconnected the service and the pictures started showing up :D

**Comment by IpsmLorem** [comment]
What was exactly the network problem ?

**Comment by Eternity512** [comment]
My ISP blocks some sites and I used a "stand-alone way to bypass DPI" to circumvent them. Using the GoodbyeDPI program, it was possible to access a "blocked" site without using a VPN.

Recently, the provider blocked the possibility of circumvention of the DPI and introduced a full scan of the DPI inside the proxy, in the role of which my computer acted. I turned off the DPI substitution service and it all worked.
